UnityProject/Assets/Scripts/Messages/Server/PossessAndUnpossess.cs

```diff
@@ -0,0 +1,57 @@
+using System.Collections;
+using System.Collections.Generic;
+using Mirror;
+using UnityEngine;
+
+namespace Messages.Server
+{
+	public class PossessAndUnpossess : ServerMessage<PossessAndUnpossess.NetMessage>
```

Comment:
```suggestion
	public class PossessAndUnpossessMessage : ServerMessage<PossessAndUnpossess.NetMessage>
```

All messages have Message at the end just to know what they are.

Might be better to also have a utility class called PossessAndUnpossess and have some nice named functions with summaries. Instead of just PossessAndUnpossess.Send()